src/Symfony/Component/Form/Extension/Core/Type/BaseType.php

```diff
@@ -75,11 +75,6 @@ public function buildView(FormView $view, FormInterface $form, array $options):
             $id = \is_string($options['form_attr']) ? $options['form_attr'] : $name;
             $fullName = $name;
             $uniqueBlockPrefix = '_'.$blockName;
-
-            // Strip leading underscores and digits. These are allowed in
-            // form names, but not in HTML4 ID attributes.
-            // https://www.w3.org/TR/html401/struct/global#adef-id
-            $id = ltrim($id, '_0123456789');
```

Comment:
should be kept IMO as written in https://github.com/symfony/symfony/issues/53976#issuecomment-2268574523

Reply:
If this change is kept (which I think is good - see my answer at https://github.com/symfony/symfony/issues/53976#issuecomment-2268688570 ), then it's a BC break and therefore should target v8.0.